### ISSUE TITLE
Low contrast (unreadable) menu in private mode, when iOS Reduce Transparency and Increase Contrast options enabled 

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#4952]
Logged via https://bugzilla.mozilla.org/show_bug.cgi?id=1550125

Steps to reproduce:

1. Install Firefox 16.2 (14898) on iOS 12.2
2. In the iOS accessibility settings (General > Accessibility), enable "Reduce Transparency" and "Increase Contrast"
3. Open Firefox, switch to private/incognito mode
4. Open the Firefox menu

Actual results:

The contrast between menu text and menu background is extremely low, to the point of being illegible. 

The attached screenshots hopefully show the effect of turning each of the two options "Reduce Transparency" (RT) and "Increase Contrast" (IC) on and off, respectively. When both enabled, the menu is completely illegible

Expected results:

The menu text should remain visible, having enough contrast with its background, regardless of iOS settings. But moreover, when accessibility-focused options are enabled.

- Increase Contrast-ON, Reduce Transparency-ON
![IC-ON-__RT-ON-](https://user-images.githubusercontent.com/32130829/57541799-86f48980-7358-11e9-9b11-13122f172c07.png)

- Increase Contrast-OFF, Reduce Transparency-ON
![IC-OFF__RT-ON-](https://user-images.githubusercontent.com/32130829/57541827-970c6900-7358-11e9-9078-b8d3db15d683.png)

- Increase Contrast-OFF, Reduce Transparency-OFF
![IC-OFF__RT-OFF](https://user-images.githubusercontent.com/32130829/57541905-c4f1ad80-7358-11e9-9046-e35b0884ccad.png)

- Increase Contrast-ON, Reduce Transparency-OFF
![IC_ON-__RT-OFF](https://user-images.githubusercontent.com/32130829/57541911-ccb15200-7358-11e9-8f6a-e356100cac3a.png)

**Comment by nil4** [comment]
@SimonBasca @garvankeeley the commit that closed this issue does not seem related to the problem reported here, i.e. lack of contrast between text and background in the Firefox menu. Is the closing of this issue intended?

**Comment by garvankeeley** [comment]
Oops, wrong commit number, thanks.

**Comment by peterkos** [comment]
I'd like to take this on if nobody has! (Looks like its still an issue in 18.1, 15719)

**Comment by garvankeeley** [comment]
Any help here would be appreciated, I am not sure how to fix this easily.
